src/ExpressionEvaluator/Core/Source/ExpressionCompiler/DkmUtilities.cs

```diff
@@ -45,17 +45,23 @@ private static IEnumerable<DkmClrModuleInstance> GetModulesInAppDomain(this DkmC
 
         internal unsafe static ImmutableArray<MetadataBlock> GetMetadataBlocks(this DkmClrRuntimeInstance runtime, DkmClrAppDomain appDomain)
```

Comment:
Agree. Thanks!

---
In reply to: [127064238](https://github.com/dotnet/roslyn/pull/20834#discussion_r127064238) [](ancestors = 127064238)